pandas/tests/frame/test_convert_to.py

```diff
@@ -229,3 +229,12 @@ def test_to_records_datetimeindex_with_tz(self, tz):
 
         # both converted to UTC, so they are equal
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_to_series(self, capsys):
+        df = DataFrame({'A': [np.nan, 2.0, np.nan]})
+        s = Series([1, 1, 1])
+
+        df.clip_lower(s, axis=0)
```

Comment:
you can use `with tm.assert_produces_warning(None):` around this to assert no warning raised rather than reading stderr